src/generator/AutoRest.Go/Templates/VersionTemplate.cshtml

```diff
@@ -24,18 +24,32 @@ const (
     // Always begin a "tag" with a dash (as per http://semver.org)
     tag   = "-beta"
 
-    semVerFormat = "%s.%s.%s%s"
     userAgentFormat = "Azure-SDK-For-Go/%s arm-%s/%s"
 )
 
+var userAgent string
```

Comment:
Fair enough. Personally, what I really wanted was the VB.NET version of static where you can define fields inside of methods so that they aren't in global scope, but are persistent between calls to that method.